pkg/handler/handler.go

```diff
@@ -564,6 +564,7 @@ func (h handler) apbAddSpec(w http.ResponseWriter, r *http.Request, params map[s
 		writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: "Invalid parameter yaml"})
 		return
 	}
+	spec.Image = spec.FQName
```

Comment:
Feels weird to put this in the handler. I would put it in `AddSpec` of `broker.go`.

Reply:
+1 putting it in AddSpec makes more sense because that's what gets called when you push to the dev endpoint.